openedx/core/djangoapps/credit/tasks.py

```diff
@@ -208,17 +216,23 @@ def _get_proctoring_requirements(course_key):
     # process
     from edx_proctoring.api import get_all_exams_for_course
 
-    requirements = [
-        {
-            'namespace': 'proctored_exam',
-            'name': exam['content_id'],
-            'display_name': exam['exam_name'],
-            'criteria': {},
-        }
-        for exam in get_all_exams_for_course(unicode(course_key))
-        # practice exams do not count towards eligibility
-        if exam['is_proctored'] and exam['is_active'] and not exam['is_practice_exam']
-    ]
+    requirements = []
+    for exam in get_all_exams_for_course(unicode(course_key)):
+        if exam['is_proctored'] and exam['is_active'] and not exam['is_practice_exam']:
+            try:
+                usage_key = UsageKey.from_string(exam['content_id'])
+                proctored_block = modulestore().get_item(usage_key)
```

Comment:
Yes we could include that but it will increase the scope of this story as:

1) We need to add a new column in https://github.com/edx/edx-proctoring repo which would leads to adding new migration.
2) Send start date from `contentstore.proctoring.register_proctored_exams` proctoring API.
3) API's in https://github.com/edx/edx-proctoring repo needs to get updated so that we could add and get start date from api call.
4) Point 2 could also lead us to some failing tests which we need to deal with and also may be we have to write some new tests as well.